packages/amplify-e2e-tests/src/__tests__/schema-iterative-rollback-1.test.ts

```diff
@@ -47,17 +47,17 @@ describe('Iterative Rollback - add 2 @keys ', () => {
 
     const finalSchema = path.join('iterative-push', 'two-key-add', 'final-schema.graphql');
     updateApiSchema(projectDir, apiName, finalSchema);
-    await cancelIterativeAmplifyPush(projectDir, { current: 2, max: 3 });
+    await cancelIterativeAmplifyPush(projectDir, /.*UPDATE_COMPLETE RecordTable*/);
 
     await new Promise(res => setTimeout(res, 1000));
 
     table = await getDDBTable(tableName, region);
     expect(table.Table.GlobalSecondaryIndexes).toBeDefined();
     expect(table.Table.GlobalSecondaryIndexes.length).toEqual(1);
 
-    await amplifyPushIterativeRollback(projectDir);
+    // await amplifyPushIterativeRollback(projectDir);
```

Comment:
Do you just want to remove these or are you working on something with CCI?

Reply:
No I need those in there, adding them back since it was commented out for testing.